Dockerfile

```diff
@@ -45,6 +45,7 @@ COPY . /code/
 # copy built react and node libraries for production mode
 COPY --from=node-prod-deps /usr/src/app/package-lock.json /code/package-lock.json
 COPY --from=node-prod-deps /usr/src/app/webpack-stats.json /code/webpack-stats.json
+COPY --from=node-prod-deps /usr/src/app/assets/images /code/static
```

Comment:
Images can be contained in the `assets` directory instead of `dashboard/static` director